apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -26,6 +26,7 @@ type DatadogAgentStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:unservedversion
```

Comment:
Just to confirm: adding this comment is changing the status of `served` in the CRD yaml file definition?

Reply:
does it allows us to release a version from `main` without exposing v2alpha1 to user?

Reply:
Yes, correct. So `v2alpha1` will be present in the CRD but it's unusable. The other option is to flag `v2alpha1` as `skipped` in which case it's not generated at all.